spacy_transformers/data_classes.py

```diff
@@ -177,8 +180,10 @@ def zeros(cls, length: int, width: int, *, xp=numpy) -> "TransformerData":
         )
 
     @property
-    def tensors(self) -> Tuple[Union[FloatsXd, List[FloatsXd]]]:
-        return self.model_output.to_tuple()
+    def tensors(self, exlude=EXCLUDED_KEYS) -> Tuple[Union[FloatsXd, List[FloatsXd]]]:
+        return tuple(
+            self.model_output[k] for k in self.model_output.keys() if k not in exlude
+        )
```

Comment:
This particular change isn't strictly necessary, but I thought it would keep the API consistent (same comment for similar change in the same file)